### DISCUSSION TITLE
Erro ao criar novo repositório

### DISCUSSION THREAD

**Discussion by AgnaldoVaz** [community/community#69183] [New to GitHub]
Estou tendo um problema quando tento criar um novo repositório, quando começo a digitar o nome do repositório aparece a seguinte mensagem de erro:  Parece que algo deu errado!
alguém poderia me ajudar a resolver esse erro?

**Comment by queenofcorgis** [comment]
Hi there @AgnaldoVaz!

* Are you still experiencing this? If so, have you tried creating it with a different page or using a different browser?
* Please keep in mind we take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.
